code/game/gamemodes/changeling/powers/shriek.dm

```diff
@@ -1,6 +1,6 @@
 /datum/power/changeling/resonant_shriek
 	name = "Resonant Shriek"
-	desc = "Our lungs and vocal chords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded biologicals and synthetics."
+	desc = "Our lungs and vocal cords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded."
```

Comment:
'biologicals' refers to medications, and 'organics' would apply to the 'ling itself.